meson-mode.el

```diff
@@ -110,6 +110,12 @@
       (or (eval `(or ,@meson-builtin-vars)))
       symbol-end))
 
+(defconst meson-vars-regexp
```

Comment:
I'd prefer this to be called `meson-var-assign-regexp`.

Reply:
Done!